rules/renaming/tests/Rector/MethodCall/RenameMethodRector/RenameMethodRectorTest.php

```diff
@@ -40,6 +40,7 @@ protected function getRectorsWithConfiguration(): array
                     new MethodCallRename(AbstractType::class, 'setDefaultOptions', 'configureOptions'),
                     new MethodCallRename(Html::class, 'add', 'addHtml'),
                     new MethodCallRename('*Presenter', 'run', '__invoke'),
+                    new MethodCallRename('DemoFile', 'run', 'test'),
```

Comment:
Here must be fqn class name, with namespace

Reply:
So it's just not working in the Playground because it doesn't have a namespace?

Reply:
Looks like that :+1: